src/components/interfaces/MOBILE_API.xml

```diff
@@ -3131,6 +3445,15 @@
                 If included, the platform supports seat controls.
             </description >
         </param>
+        <param name="audioControlCapabilities" type="AudioControlCapabilities" mandatory="false" minsize="1" maxsize="100" array="true">
```

Comment:
Still missing `since`

Reply:
@jacobkeeler Fixed at commit 593f38dd0a9cfb336623c77fa160eb6f871f806e

Reply:
Still missing for these parameters